openapi3.yaml

```diff
@@ -212,6 +431,45 @@ components:
           type: string
         dateCreated:
           type: string
+    dataStores:
+      type: array
+      items:
+        $ref: '#/components/schemas/dataStore'
+    dataStore:
+      type: object
+      properties:
+        name:
+          type: string
+        link:
+          type: string
+    dataStoreInfo:
+      type: object
+      properties:
+        name:
+          type: string
+        dateCreated:
+          type: string
+        host:
+          type: string
+        port:
+          type: number
+        schema:
+          type: string
+        dbType:
+          type: string
+          enum: [postgis]
+        dbName:
+          type: string
+        sslMode:
+          type: string
+          enum: [DISABLE, ALLOW, PREFER, REQUIRE, VERIFY_CA, VERIFY_FULL]
```

Comment:
added validations. Added them in the index.ts file after some consult with ofer. it will crash the app as it should because we dont want to container to go up when the config are incorrect

Reply:
added a logger inside the function and tests